test/built-ins/TypedArray/prototype/at/prop-desc.js

```diff
@@ -17,6 +17,8 @@ assert.sameValue(
   'The value of `typeof TypedArray.prototype.at` is "function"'
 );
 
-verifyNotEnumerable(TypedArray.prototype, 'item');
-verifyWritable(TypedArray.prototype, 'item');
-verifyConfigurable(TypedArray.prototype, 'item');
+verifyProperty(TypedArray.prototype.at, 'name', {
```

Comment:
Oops, this and the String one are mistakenly checking `name` here. Will submit another PR to fix.